tc/aten/aten_compiler.h

```diff
@@ -22,9 +22,9 @@
 #include <ATen/ATen.h>
 #include <ATen/DLConvertor.h>
 
-#include "tc/aten/utils.h"
-#include "tc/core/execution_engine.h"
-#include "tc/lang/parser.h"
+#include "aten/utils.h"
+#include "core/execution_engine.h"
```

Comment:
Actually having includes starting from "tc", in my opinion, is what we should be targeting. this way we provide a proper isolation between tc and any other libraries when TC is included as a submodule or is included in an FBCode fashion. 


Also number of changed files would be significantly smaller in this case :)

Reply:
I agree. I'll address this and update PR. There will be cmake change needed only for this then. Thanks.